crates/snp_validation/src/config.rs

```diff
@@ -0,0 +1,179 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+use crate::{PlatformInfoFlags, PolicyFlags};
+use flagset::FlagSet;
+use semver::Version;
+use serde::de::Error;
+use serde::{Deserialize, Deserializer, Serialize};
+use std::str::FromStr;
+
+#[derive(Clone, Deserialize, Debug, Default, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct Config {
+    /// Values for `author_key_digest` in the report body.
+    /// This is the list of public keys which have signed the signature of the Enarx binary.
+    #[serde(default)]
+    #[serde(deserialize_with = "from_hex")]
+    pub enarx_signer: Option<Vec<Vec<u8>>>,
```

Comment:
Why the rename to `enarx_signer` ?? It is the key, which signed the key, which signed enarx.

Reply:
Ah, ok, to have the same name as the one in SGX

Reply:
So, SGX `enarx_signer` and SNP `id_key` are the same then. SNP `enarx_signer` signed SNP `id_key`.

Reply:
Also, do we want the name `enarx` in here, or can steward attest to other software as well?

Reply:
Good question…

Reply:
Enarx name removed